### PR TITLE
Mixer naming cleanup

### PR DESCRIPTION
**Pull request by anton-matosov**

### CODE REVIEW

**Comment by LorenzMeier** [comment]
@anton-matosov If you can confirm that the FMU interface works and IO still works too, I can merge all these PRs.

**Comment by anton-matosov** [comment]
I will perform some extra testing tomorrow and will post a confirmation then.

**Comment by anton-matosov** [comment]
I have just noticed that there are 2 firefly branches:
- FireFly6
- firefly (rebased FireFly6 w/o one merge commit)
What is the 'actual' one? 
I am asking this because I have based my work on the FireFly6 branch.
Can you delete the obsolete one to not bring confusion, please?

**Comment by LorenzMeier** [comment]
The one the PR is firefly, and I've deleted the FireFly6 already yesterday

**Comment by anton-matosov** [comment]
OK, will rebase to the new branch.

**Comment by anton-matosov** [comment]
I have done the testing (and fixed couple of issues) and confirm that both FMU and IO outputs are working perfectly fine.

File: ROMFS/px4fmu_common/init.d/rc.interface [hunk 1]
```diff
@@ -23,7 +37,7 @@ then
 	else
 		set OUTPUT_DEV /dev/pwm_output
 	fi
-
+
```
Comment (LorenzMeier) [inline ROMFS/px4fmu_common/init.d/rc.interface]: whitespace

File: ROMFS/px4fmu_common/init.d/rc.interface [hunk 2]
```diff
@@ -80,3 +94,34 @@ then
...
+	if [ $MIXER_AUX_FILE != none -a $FMU_MODE == pwm ]
+	then
+		if fmu mode_pwm
+		then
+			mixer load /dev/pwm_output1 $MIXER_AUX_FILE
+		else
+			tone_alarm $TUNE_ERR
+		fi
+	fi
+fi
```
Comment (LorenzMeier) [inline ROMFS/px4fmu_common/init.d/rc.interface]: missing end of line

File: ROMFS/px4fmu_common/init.d/1003_rc_quad_+.hil [hunk 3]
```diff
@@ -7,6 +7,6 @@
 
 sh /etc/init.d/rc.mc_defaults
 
-set MIXER FMU_quad_+
+set MIXER quad_+
 
 set HIL yes
```
Comment (LorenzMeier) [inline ROMFS/px4fmu_common/init.d/1003_rc_quad_+.hil]: end of file

File: ROMFS/px4fmu_common/init.d/2101_hk_bixler [hunk 4]
```diff
@@ -2,4 +2,4 @@
 
 sh /etc/init.d/rc.fw_defaults
 
-set MIXER FMU_AERT
\ No newline at end of file
+set MIXER AERT
```
Comment (LorenzMeier) [inline ROMFS/px4fmu_common/init.d/2101_hk_bixler]: end of file

**Comment by LorenzMeier** [comment]
Great, thanks! I've added a few comments. Would you mind addressing them and rebasing on master? Thanks!

File: ROMFS/px4fmu_common/init.d/1003_rc_quad_+.hil (hunk 3 above)
Comment (anton-matosov) [inline ROMFS/px4fmu_common/init.d/1003_rc_quad_+.hil]: Fixed

File: ROMFS/px4fmu_common/init.d/rc.interface (hunk 2 above)
Comment (anton-matosov) [inline ROMFS/px4fmu_common/init.d/rc.interface]: fixed

File: ROMFS/px4fmu_common/init.d/2101_hk_bixler (hunk 4 above)
Comment (anton-matosov) [inline ROMFS/px4fmu_common/init.d/2101_hk_bixler]: fixed

File: ROMFS/px4fmu_common/init.d/rc.interface (hunk 1 above)
Comment (anton-matosov) [inline ROMFS/px4fmu_common/init.d/rc.interface]: fixed

**Comment by anton-matosov** [comment]
Done

**Comment by LorenzMeier** [comment]
Nice! Will merge once it passes.

**Pull request merged**